packages/cli/src/constructs/check-group.ts

```diff
@@ -166,7 +170,8 @@ export class CheckGroup extends Construct {
     // `frequency` is not a CheckGroup resource property. Not present in synthesize()
     this.frequency = props.frequency
     this.apiCheckDefaults = { ...defaultApiCheckDefaults, ...props.apiCheckDefaults }
-
+    this.alertSettings = props.alertEscalationPolicy
+    this.useGlobalAlertSettings = !this.alertSettings
```

Comment:
@tnolet what do you think about this? Basically we set `useGlobalAlertSettings` to `true` if `alertSettings` doesn't exist and vice versa

Reply:
@umutuzgur @ferrandiaz hmmmm, feels a bit iffy / hidden magic. Not sure if we have a great alternative.

Reply:
Just my 2 cents, but I think it's nice. As a user, I think that it would be confusing if I passed `alertSettings`, but they weren't actually applied because I need to also explicitly set `useGlobalAlertSettings`. I'm not sure there's a case in the CLI were I would pass `alertSettings` but not want it to actually be used.

Reply:
@clample yes, I agree. This seems to be the best UX. @umutuzgur @ferrandiaz let's go with this.